app/assignment/directives/instructorAssignmentTable.js

```diff
@@ -230,15 +230,22 @@ let instructorAssignmentTable = function ($rootScope, AssignmentActionCreators,
 										var sectionGroup = scope.view.state.sectionGroups.list[sectionGroupId];
 										var displayTitle = "";
 										var plannedSeatsHtml = "";
-										var unitsLow = "";
+										var units = "";
 
 										var course = scope.view.state.courses.list[sectionGroup.courseId];
 										var subTitle = course.title;
 										displayTitle += course.subjectCode + " " + course.courseNumber + "-" + course.sequencePattern;
 
 										var plannedSeats = sectionGroup.plannedSeats || "0";
 										plannedSeatsHtml = "<small>Seats: " + plannedSeats + "</small>";
-										unitsLow = "<small>Units: " + course.unitsLow + "</small>";
+
+										if (sectionGroup.unitsVariable) {
+											units = "Units (Variable): " + sectionGroup.unitsVariable;
+										} else if (course.unitsHigh) {
```

Comment:
We mess up this logic a lot, so I would instead add a transient helper on the entity in the backend so you just see a boolean property isVariableUnit or something like that, and then make use of that here.